.github/CODEOWNERS

```diff
@@ -68,7 +68,7 @@
 /typings/ @vojtechjelinek
 /tsconfig.json @vojtechjelinek
 /tsconfig-strict.json @vojtechjelinek
-
+/parser-tsconfig.eslint.json @vojtechjelinek
```

Comment:
Please assign it to the dev-workflow team as well (You can assign @Hudda and @DubeySandeep)!

Reply:
Done